docs/guide/Titanium_SDK/Titanium_SDK_Guide/Contributing_to_Titanium/How_to_Report_a_Bug_or_Make_a_Feature_Request/README.md

```diff
@@ -111,7 +111,7 @@ A good read on this is Eric Lippert's [How to debug small programs](http://ericl
 
 Assuming you have consulted the community via the Q&A, and you have your [test case](#CreatingaTestCase) to hand, you are ready to create a ticket:
 
-* open the JIRA [Titanium Community](http://jira.appcelerator.org/browse/TC) project page. See the [#Projects Overview](#ProjectsOverview) section for more information about the projects.
+* open the JIRA [Titanium Community](https://jira-archive.titaniumsdk.com/TC) project page. See the [#Projects Overview](#ProjectsOverview) section for more information about the projects.
```

Comment:
I don't think there's ever been a TC project. I've always known it to be AC.

Anyways, this link needs to be https://github.com/tidev/titanium_mobile/issues and update the wording from JIRA to GitHub issue.

Reply:
There are tons of JIRA mentions. That should be handled in another PR

Reply:
Sorry, yes there are tons of JIRA mentions, but in this case, I'm specifically referring to this link. We don't want to say create a JIRA ticket by going to GitHub issues.

Reply:
Fixed that